src/ref_db.h

```diff
@@ -0,0 +1,31 @@
+#ifndef GODOT_JVM_REF_DB_H
+#define GODOT_JVM_REF_DB_H
+
+#include <cstdint>
+#include "core/hash_map.h"
+#include "core/reference.h"
+#include "core/list.h"
+#include "core/os/mutex.h"
+
+class RefDB {
```

Comment:
Missing delete of copy constructor and copy assignement operator on this meyer singleton.